seacatauth/openidconnect/handler/userinfo.py

```diff
@@ -55,8 +55,6 @@ async def userinfo(self, request):
 			session,
 			tenant=request.query.get("tenant", "*")
 		)
-		if userinfo["result"] == "CREDENTIALS-NOT-FOUND":
-			return self.error_response("invalid_credentials", "Invalid credentials.")
```

Comment:
Is there a check for incorrect userinfo then?

Reply:
at this moment userinfo is always created from session data. and session cannot be created with invalid credentials.

so this check is redundant, at least for now.